frontend/src/stories/ButtonArray.stories.jsx

```diff
@@ -58,7 +58,6 @@ export const CategorizationWithHiddenText = {
             view: "BUTTON_ARRAY",
             explainer: "",
             question: "",
-            result_id: 16549,
```

Comment:
Was the `result_id` removed because it's not necessary? I thought it could still be passed in a `Question` type?

Reply:
I also removed it from the Trial props, was that unnecessary?

Reply:
It's not necessary in the `Trial` props, but will be necessary in the `Question` props, and this is a `Question`, right? For storybook, it won't matter, but as far as I'm aware it's still a valid prop here.

Reply:
I've re-added it, also as an optional `Question` interface property